CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing
 
-If you would like to contribute code you can do so through GitHub by first forking and cloning this repository, then sending a pull request.
+If you would like to contribute code you can do so through GitHub by first forking then cloning this repository, then sending a pull request.
```

Comment:
This would be a good place to link the Atlassian guide. Maybe something the lines of:
"You can visit with page to learn more about the Forking Workflow"